src/oatpp/core/data/stream/Stream.cpp

```diff
@@ -399,7 +400,7 @@ v_io_size ConsistentOutputStream::writeAsString(v_uint64 value){
 
 v_io_size ConsistentOutputStream::writeAsString(v_float32 value){
   v_char8 a[100];
-  auto size = utils::conversion::float32ToCharSequence(value, &a[0], 100);
+  auto size = parser::json::Utils::float32ToJson(value, &a[0], 100);
```

Comment:
Since `ConsistentOutputStream` is a general-purpose class - it should
use `auto size = utils::conversion::float32ToCharSequence(value, &a[0], 100);` as before